metaflow/plugins/aws/batch/batch_client.py

```diff
@@ -282,6 +287,29 @@ def _register_job_definition(
                     {"sourceVolume": name, "containerPath": host_path}
                 )
 
+        if use_tmpfs or tmpfs_size:
+            if tmpfs_size:
```

Comment:
there should probably be a check that the final value for tmpfs_size is less than `memory=`, because otherwise tasks run into
```
Error: [Errno 12] Cannot allocate memory
```

Reply:
also a bit unsure on the exact constraints of `tmpfs_size`, as setting this to 1000 and trying to S3 get a 1gb file results in a
```
Error: [Errno 28] No space left on device
```
but somewhat expected that redlining the attached storage would run out of space due to possible overhead

Reply:
I tried setting `@batch(memory=10000, tmpfs_size=120000)` and surprisingly it creates a tmpfs volume of the requested size. It'd probably fail allocating more than 10GB though. I think it's ok to leave it as-is for now.

Storing more data than what's set by `tmpfs_size` results to `No space left` as you said, which is expected behavior